packages/app/src/containers/SwatchesToolbar.tsx

```diff
@@ -0,0 +1,47 @@
+import { Spacer } from 'noya-designsystem';
+import {
+  PlusIcon,
+} from '@radix-ui/react-icons';
+import { useMemo } from 'react';
+import styled, { useTheme } from 'styled-components';
+import Button from 'noya-designsystem/src/components/Button';
+import { useApplicationState } from '../contexts/ApplicationStateContext';
+
+const Container = styled.header(({ theme }) => ({
+  height: `${theme.sizes.toolbar.height}px`,
+  display: 'flex',
+  borderBottom: `1px solid ${theme.colors.dividerStrong}`,
+  alignItems: 'center',
+  backgroundColor: theme.colors.sidebar.background,
+  color: theme.colors.textMuted,
+}));
+
+export default function Toolbar() {
+  const [state, dispatch] = useApplicationState();
+  const itemSeparatorSize = useTheme().sizes.toolbar.itemSeparator;
+  const interactionType = state.interactionState.type;
+
+  return useMemo(
+    () => (
+      <Container>
+        <div>
```

Comment:
We can remove the title, we don't need it yet, and it may go somewhere else

Reply:
Oh! I left it for now 😅